### PR TITLE
rewrite config initial phase in python

### PR DESCRIPTION
**Pull request by rjonczy**
## Purpose

Generate config for non-standard networks in python (initial phase)

## How does the solution address the problem

Changed from bash into python part of config generation.

## Changes made

Changed config generation for:
- initial validators

### CODE REVIEW

File: charts/taraxa-node/templates/initconfig-consensus-node.yaml [hunk 1]
```diff
@@ -11,6 +11,85 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 data:
+  genconfig.py: |-
+    import json
+    import sys
+    import subprocess
+
+    def get_vrf_public(vrf_prv_key):
```
Comment (MatusKysel) [inline charts/taraxa-node/templates/initconfig-consensus-node.yaml:19]: vrf public key is just a part of private key, so you could just split private key. 
```
	"vrf_public" : "42f6022cce105e9f251e8054378cf1f435ea1e7bd0ba12b5b22c48a8c68d7db0",
	"vrf_secret" : "84117a0e78a6caadf2ba67414d4310136feddff3abbf7acc80b6ed9912a8ec8942f6022cce105e9f251e8054378cf1f435ea1e7bd0ba12b5b22c48a8c68d7db0"
```

you can see that public key private_key[64:]

**Review by MatusKysel — APPROVED** [review]

File: charts/taraxa-node/templates/initconfig-consensus-node.yaml (hunk 1 above)
Comment (rattrap) [reply charts/taraxa-node/templates/initconfig-consensus-node.yaml:19]: Nice. Didn't know that

**Review by rattrap — APPROVED** [review]
LGTM

File: charts/taraxa-node/templates/initconfig-consensus-node.yaml (hunk 1 above)
Comment (rjonczy) [reply charts/taraxa-node/templates/initconfig-consensus-node.yaml:19]: @MatusKysel thanks. Now i merge this as I have another issue:

https://github.com/Taraxa-project/taraxa-node/issues/1906

In that issue I will completely change config generation into python and change the way how i obtain public key.

**Pull request merged**